build.gradle.kts

```diff
@@ -43,22 +50,26 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
+        kotlin.srcDir("build/generated/ksp/metadata/commonMain/kotlin")
```

Comment:
Thanks @chiroptical, Do you need this if you add a package in the Sample file?

Reply:
Sorry, what do you mean by "Sample file"? I am pretty sure that is how gradle/IDE finds all of the KSP generated companion objects. TBH, I am not super familiar with gradle and there may be a better way to do this. I just don't have enough Kotlin experience to help there.

This is working on my machine. I can use the optics within and outside `commonMain` with this.